.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -153,7 +153,7 @@ AddType text/vtt                            vtt
 
     # Redirects for "latest" version
     RewriteRule ^(container-platform|enterprise)/?$ /container-platform/latest [R=301]
-    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.14/$2 [NE,R=301]
+    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.15/$2 [NE,R=301]
```

Comment:
@bscott-rh, will you please confirm the redirects?

Reply:
Redirects LGTM